README.md

```diff
@@ -16,72 +16,77 @@ You can simply install it via opam in your current switch by running:
 opam install opam-monorepo
 ```
 
-Note that once it is installed you can invoke it as an opam command:
+Note that once it's installed you can invoke it as an opam command:
```

Comment:
Out of curiosity, what's the rule about contraction in written English?

I personally use contractions when I write but I remember somebody telling me it shouldn't be used in written English. Seeing this, I'm now confused!

Reply:
I prefer them because it's more casual in most cases. The tone of our documentation, by directly talking to the reader in the second person (you, your) and us in the first person (we, our), I feel it's more natural to use contractions unless there needs to be particular emphasis.

Not using contractions gives it a more formal tone. 

Basically, it's a matter of preference. Both are correct.